src/Apps/Components/Layouts/index.tsx

```diff
@@ -16,12 +16,17 @@ export interface LayoutProps extends BaseLayoutProps {
   variant?: LayoutVariant
 }
 
+const LayoutLogoOnlyFullBleed = ({ children }: BaseLayoutProps) => (
```

Comment:
Please move to own file like the other layouts, and https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-implicit-returns-on-react-components